tests/components/ct-vue-vite/tests/events/events.spec.js

```diff
@@ -0,0 +1,16 @@
+import { test, expect } from '@playwright/experimental-ct-vue';
```

Comment:
I don't think we need to go all the way in ts/js and tsx/jsx parity. It would be sufficient to check that all notations work, but no need for functional changes for all of them.

Reply:
Thank you for the feedback! 

I'm not completely sure how you envision it. Was thinking of keeping all the `TS`/`TSX` tests both for documentation and to make sure everything works as expected. Perhaps keep `render.js` tests to make sure that works as expected as well?

Reply:
BTW. there are also other ways to speed up the pipeline if they need to be removed for performance reasons. Like using pnpm and or maybe a tool like turborepo. When one of the component test projects is modified, the entire pipeline is executed, which i don't think is necessary.

Reply:
We don't use tests for documentation - doing that typically compromises both in the end. So we should have tests for coverage and examples for the documentation. Since this is about the tests, all we care about is coverage. Which means we don't need to repeat all the tests in all the dialects, unless that triggers paths we want to test.

Reply:
Considering your router tests as an example - once we've established that we can send hooks config in both JS and JSX notations, we don't need to test the routes twice. I.e. testing that config is plumbed in both notations and then testing routes using only one notation is sufficient. It is not about the speed of execution, more about the identical tests diverging over time.

Happy New Year btw!

Reply:
Thank you for the replies! Happy new year!